abis/linux/x86_64/signal.h

```diff
@@ -0,0 +1,108 @@
+#ifndef _ABIBITS_SIGNAL_H
+#define _ABIBITS_SIGNAL_H
+
+#ifdef __MLIBC_POSIX_OPTION
+#include <bits/posix/siginfo_t.h>
```

Comment:
siginfo_t is also a OS-specific ABI (?).

Reply:
siginfo is needed for sigval